app.rb

```diff
@@ -1,7 +1,36 @@
 require 'sinatra/base'
+require 'sinatra/reloader'
+require_relative './lib/game'
+require_relative './lib/player'
+
 class RockPaperScissors < Sinatra::Base
-  get '/test' do
-    'test page'
+
+  get '/' do
+    erb :welcome 
+  end
+
+  post '/name' do 
+    $player = params[:player]
+    #$player = Player.new(params[:player])
+    redirect '/play'
+  end 
+
+  get '/play' do 
+    @player = $player
+   erb :play 
+  end 
+
+
+  post '/choserock' do
```

Comment:
Have one page for your result, rather than 3

Reply:
Have now changed this to be one page